serverless/event_sources/serverless-kafka-source.adoc

```diff
@@ -0,0 +1,15 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-kafka-source"]
+= Using a Kafka source
+include::modules/common-attributes.adoc[]
+:context: serverless-kafka-source
+toc::[]
+
+[id="serverless-kafka-source-prerequisites"]
```

Comment:
The same for this one, does it need `_{context}` or is that not relevant for titles in assembly files?

Reply:
Just for modules afaik